docResources/Features.md

```diff
@@ -59,11 +59,11 @@
     | `Text`        | :white_check_mark: |
     | `TextField`   | :white_check_mark: |
     | `Toggle`      | :white_check_mark: |
-    | Gradients     |  |
+    | `Gradients`   |  |
     | `Link`        |  |
     | `Path`        |  |
     | `SecureField` |  |
-    | Shapes        |  |
+    | `Shapes`      | :white_check_mark: |
```

Comment:
The reason I didn't use a `code block` for this and Gradients, etc. is because those are just a reference to the different views, but the names are not an actual component in itself. For Example, you can't find a `Shapes` type nor a `Gradients`. Instead you have `Rectangle`, `Capsule`, so on
Maybe using something like: | `Shape` views | may be better?